mpmp/src/model/Player.java

```diff
@@ -101,6 +104,37 @@ public void ragequit() {
 	}
 
 	/**
+	 * Add a unjail card to the player.
+	 */
+	public void addUnjailCard() {
+		unjails++;
+	}
+
+	/**
+	 * Give one unjail card to player p. Usually against monetary compensation.
+	 */
+	public void giveUnjailCard(Player p) {
+		if(unjails <= 0)
+			return;
+
+		unjails--;
+		p.unjails++;
+	}
+
+	/**
+	 * Leave the prison by using a unjail card.
+	 */
+	public void useUnjailCard() {
```

Comment:
Maybe. My rationale is that giving someone a card or using one should not be possible in the interface if you don't have one, so there's no incentive to even check the error if it will never happen. I could certainly add a log line.